api/src/main/scala/uk/ac/wellcome/platform/api/finatra/exceptions/ElasticsearchResponseExceptionMapper.scala

```diff
@@ -29,52 +29,91 @@ class ElasticsearchResponseExceptionMapper @Inject()(
     error(
       s"Sending HTTP 400 from ElasticsearchResponseExceptionMapper ($message)",
       exception)
-    DisplayError(Error(variant = "http-400", description = Some(message)))
+    DisplayError(Error(variant = s"http-400", description = Some(message)))
```

Comment:
s-strings here are unnecessary, and an artefact of an earlier version of this code